docs/src/modules/java-protobuf/pages/views.adoc

```diff
@@ -321,25 +321,54 @@ include::example$scala-protobuf-eventsourced-customer-registry/src/main/scala/cu
 [#topic-view]
 == Creating a View from a topic
 
-The source of a View can be an eventing topic. You define it in the same way as shown in <<event-sourced-entity>> or <<value-entity>>, but leave out the `eventing.in` annotation in the Protobuf file.
+The source of a View can be a broker topic.
+You define it in the same way as shown in <<event-sourced-entity>> or <<value-entity>>.
+Note that, on your producer side you must manually pass the `ce-subject` metadata, required by the View component.
```

Comment:
That's frustrating, intellij editor is messing the the new lines since the latest update :/